corehq/warehouse/management/commands/create_batch.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management import BaseCommand, CommandError
+from dimagi.utils.parsing import string_to_utc_datetime
+from corehq.warehouse.models import BatchRecord
+
+
+USAGE = """Usage: ./manage.py create_batch <batch_id> -s <start_datetime> -e <end_datetime>
+"""
+
+
+class Command(BaseCommand):
+    """
+    Example: ./manage.py create_batch 222617b9-8cf0-40a2-8462-7f872e1f1344 -s 2017-05-01 -e 2017-06-01
```

Comment:
yeah this is a good point and may need some further investigation. i was under the impression that airflow handles / stores all that information. on one hand it's quite nice to be able to specify the dates because it makes the system less stateful, on the other hand it could introduce complexities on the airflow side of things. we could also make assertions about allowed date ranges based on the previous batch to make sure we don't do anything funny like skip dates